src/private/translator/avp_to_cedar.rs

```diff
@@ -39,7 +39,7 @@ impl TryFrom<PolicyDefinition> for Policy {
         match detail {
             PolicyDefinitionDetail::Static(definition_detail) => {
                 let cedar_policy = cedar_policy::Policy::parse(
-                    Some(policy_id.clone()),
+                    Some(cedar_policy::PolicyId::from_str(policy_id.as_str()).unwrap()),
```

Comment:
Don't use `.unwrap` or `.expect` outside of test code. This should handle the error by returning a `TranslatorException`

Reply:
PolicyId::from_str returns an Infallable error, which means it should never happen. I was looking up standard error handling and while .unwrap is heavily discouraged in production code, it can be used if the code will never fail. In this case, would it be ok to keep the unwrap?

Error handling references:
https://blog.burntsushi.net/unwrap/
https://users.rust-lang.org/t/best-practices-for-unwrap/101335

Reply:
Actually scratch that, this isn't accounting for future changes

Reply:
Hopefully done!